platform-specific/win-gettimeofday.c

```diff
@@ -0,0 +1,55 @@
+/* 
+ * Author: Ugo Varetto - ugovaretto@gmail.com
```

Comment:
AFAIK, the name of the contributor is Jan Roman?
Who is Ugo Varetto?
If he is a co-author, either split his contribution into a different commit/PR, with "Signed-off-by: ugovaretto@gmail.com". And ugovaretto@gmail.com needs to be used to sign the ECA.

Reply:
Sorry, I overseen that in my first review.

Reply:
The code by Ugo Varetto is actually third party code I included to allow for a reuse of the existing function implementation based on `gettimeofday()` in `dtls_time.c`. I noticed, however, that this is probably not needed and the Windows-specific time API can probably be used instead, resolving all legal matters in the process. Sorry for the inconvenience – I hope to be able to provide a rework soon.

Reply:
Third-party work requires a proper license by that third party. So please, if possible, either ask the author for permission (best ECA by him) or try to redesign it.

Reply:
I have now replaced the reimplementation of `gettimeofday` using a call of the Windows-specific `GetSystemTime` function. Things still work as before, getting rid of the additional code (and the potential legal issues).